scripts/create-zome-call.js

```diff
@@ -1,12 +1,36 @@
-const { findInstanceForAgent } = require('../src/utils/integration-testing/conductorConfig')
-const initAndGetHolochainClient = require('./holochain-client.js')
+const { connect: hcWebClientConnect } = require('@holochain/hc-web-client')
```

Comment:
These updates create a file almost identical to `scripts/holochain-client`.  Can we reference this file instead of the extra LOC?